src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -110,7 +110,7 @@ protected function extractAttributes($object, $format = null, array $context = [
 
             $attributeName = lcfirst(substr($method->name, 0 === strpos($method->name, 'is') ? 2 : 3));
 
-            if ($this->isAllowedAttribute($object, $attributeName)) {
+            if ($this->isAllowedAttribute($object, $attributeName, $format, $context)) {
```

Comment:
This change (but not the test) should probably also be made in 3.4. What do you think?

Reply:
Yes it should be in 3.4 too. (I did not do it in 3.4 because AbstractNormalizer did not contain the constant IGNORED_ATTRIBUTES but it was a mistake ..)

It is better to do a second PR? (because of the unit test)

Reply:
I think having a second (additional) PR which comes with a different test case would be great. :)

Reply:
ok , I do that now ;)

Reply:
Thank you! :)

Reply:
@xabbuh  https://github.com/symfony/symfony/pull/30720